clients/client/src/test/java/org/projectnessie/client/http/TestHttpClient.java

```diff
@@ -77,9 +78,9 @@ void testGet() throws Exception {
         h -> {
           Assertions.assertEquals("GET", h.getRequestMethod());
           String response = MAPPER.writeValueAsString(inputBean);
-          h.sendResponseHeaders(200, response.getBytes().length);
+          h.sendResponseHeaders(200, response.getBytes(StandardCharsets.UTF_8).length);
           OutputStream os = h.getResponseBody();
```

Comment:
nit: seems like all of these should use try-resources block

Reply:
Feel free to open a PR to fix it